client/app.tsx

```diff
@@ -91,6 +92,8 @@ const componentMappings = {
   'admin-share': <AdminShare crowi={crowi} />,
   'page-comments': <Comment crowi={crowi} pageId={pageId} revisionId={revisionId} revisionCreatedAt={revisionCreatedAt} isSharePage={isSharePage} />,
   'admin-page': <AdminPage crowi={crowi} />,
+
+  'help-portal': <HelpPortal crowi={crowi} />,
```

Comment:
In fact, crowi props is not used. Please remove crowi props.

Reply:
done.